lib/sandthorn/bounded_context.rb

```
@@ -0,0 +1,35 @@
+module Sandthorn
+  module BoundedContext
+
+    module ClassMethods
+      def aggregate_list()
+        @aggregate_list = p_aggregate_list(self)
+      end
+
+      private
+      def p_aggregate_list(bounded_context_module)
+        return [] unless bounded_context_module.respond_to?(:constants)
```

Shouldn't we just let this fail?